bentoml/_internal/io_descriptors/numpy.py

```diff
@@ -105,22 +103,28 @@ def predict(input_array: np.ndarray) -> np.ndarray:
 
     def __init__(
         self,
-        dtype: t.Optional[t.Union[str, "np.dtype[t.Any]"]] = None,
+        dtype: t.Optional["DTypeLike"] = None,  # type: ignore[reportUnknownParameterType]
         enforce_dtype: bool = False,
         shape: t.Optional[t.Tuple[int, ...]] = None,
         enforce_shape: bool = False,
     ):
+        import numpy as np
```

Comment:
I'm not sure if we always want to import numpy when a input/output want to use `NumpyNdarray`?

Reply:
It should be ok inside `__init__`, we just need to avoid import during initial loading of this module